rocky/OOI_database_seed.json

```diff
@@ -489,8 +489,8 @@
     "fields": {
       "last_updated": "2021-08-09T14:50:09.835Z",
       "data": {
-        "description": "A database port is open.",
-        "recommendation": "Secure this port",
+        "description": "A port commonly used by databases is open.",
+        "recommendation": "Manually validate whether this open port should actually be open.",
```

Comment:
What is the use case for having a database open to the internet? I can't think of any and it is a big security risk so I think the recommendation should be that the port be closed.

Reply:
True, we should probably at some point differentiate between ports, and actual fingerprinted services on them.